### PR TITLE
add S_NEXT_LINE flag, draw midi player on next line

### PR DESCRIPTION
**Pull request by rfomin**
Alternative to #911 
image

### CODE REVIEW

**Comment by MrAlaux** [comment]
Wouldn't this overlap with text messages that appear right at that Y position? Messages like "Value out of range", which can be triggered by two items in that page, for example _Number of Sound Channels_.

**Comment by rfomin** [comment]
> wouldn't this overlap with text messages that appear right at that Y position?

Fixed in https://github.com/fabiangreffrath/woof/pull/912/commits/79b9e724d18c1cccd834ac18e400993fc92ee66c

File: src/m_menu.c [hunk 1]
```diff
@@ -3924,8 +3935,6 @@ setup_menu_t gen_settings1[] = { // General Settings screen1
   {"Vertical Sync", S_YESNO, m_null, M_X,
    M_Y+ gen1_vsync*M_SPC, {"use_vsync"}, 0, I_ResetScreen},
 
-  {"", S_SKIP, m_null, M_X, M_Y + gen1_stub1*M_SPC},
```
Comment (fabiangreffrath) [inline src/m_menu.c:3927]: I'm a bit sad for this gap. :cry:

**Review by fabiangreffrath — APPROVED** [review]

File: src/m_menu.c (hunk 1 above)
Comment (rfomin) [reply src/m_menu.c:3927]: Maybe we can move the translucency to "Display Options", but we don't have much space here either. Also we can cut some options, for example we can move "Sound Channels" to config only. Perhaps it's time to rearrange menus again 😄

**Pull request merged**